### PR TITLE
Move at-least-node to devDependencies

### PR DESCRIPTION
**Pull request by RyanZim**
After #894 & #895

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,12 +37,12 @@
   "author": "JP Richardson <jprichardson@gmail.com>",
   "license": "MIT",
   "dependencies": {
-    "at-least-node": "^1.0.0",
     "graceful-fs": "^4.2.0",
```
Comment (JPeer264) [inline package.json:40]: Not target of this PR, but should we update `graceful-fs` (and maybe all the dev dependencies too?). Just saw that there is already v`4.2.6`. Could be a nice move during our major version bump

**Review by JPeer264 — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (RyanZim) [reply package.json:40]: The `^` will already pull in the latest version; I don't see the point of bumping it explicitly.

I'm fine with bumping dev dependencies, but no need to tie that to the major bump. Feel free to jump on that whenever you have time.

Comment (JPeer264) [reply package.json:40]: Ah ya true my bad!

**Review by manidlou — APPROVED** [review]
LGTM

**Pull request merged**